backend/curriculum_tracking/models.py

```diff
@@ -45,10 +45,10 @@ def __str__(self):
 
 class ContentItemOrder(models.Model, Mixins):
     post = models.ForeignKey(
-        "ContentItem", on_delete=models.PROTECT, related_name="pre_ordered_content"
+        "ContentItem", on_delete=models.PROTECT, related_name="pre_ordered_content", null=True
     )
     pre = models.ForeignKey(
-        "ContentItem", on_delete=models.PROTECT, related_name="post_ordered_content"
+        "ContentItem", on_delete=models.PROTECT, related_name="post_ordered_content", null=True
```

Comment:
I dont think this is wise. If there is an instance of ContentItemOrder that has `pre` or `post` as None then it means that there was a bug somewhere else. 

Data integrity is super important. That's why we have constraints. If there is a piece of code that conflcts with a constraint then it means that the code should be fixed. The database shouldn't be edited in order to allow invalid data.